zerver/models.py

```diff
@@ -602,6 +602,7 @@ class UserProfile(AbstractBaseUser, PermissionsMixin):
     # Fields from models.AbstractUser minus last_name and first_name,
     # which we don't use; email is modified to make it indexed and unique.
     email = models.EmailField(blank=False, db_index=True)  # type: str
+    actual_email = models.EmailField(blank=False, db_index=True)  # type: str
```

Comment:
What do you think about `delivery_email` for this variable name?  I'm starting to think that might be a better name.

Reply:
Sure, that makes sense.